irods_consortium_continuous_integration_test_hook.py

```diff
@@ -81,6 +55,7 @@ def main():
     os_specific_directory = irods_python_ci_utilities.append_os_specific_directory(built_packages_root_directory)
 
     irods_python_ci_utilities.install_os_packages_from_files(glob.glob(os.path.join(os_specific_directory, 'irods-resource-plugin-s3*.{0}'.format(package_suffix))))
+    install_test_prerequisites()
 
     install_build_prerequisites()
```

Comment:
I'm considering squashing the changes having to do with the test hook into one because separating them is hard. As you can see, this commit is not a valid change by itself as this function is deleted above.

Reply:
I will squash down the test hook-related changes